### ISSUE TITLE
alt missing in icon image #17358

### ISSUE THREAD

**Issue by abidkh** [microsoft/fluentui#17538]
I am using Icon of fluent UI
![image](https://user-images.githubusercontent.com/41537394/112271649-33b42780-8ca1-11eb-8b63-d272ad2dafc2.png)

While  rendering the alt is missing in this tag.And because of this we get accessibilty issues.

**Comment by abidkh** [comment]
Would u pls provide me the solution of this issues

**Comment by gouttierre** [comment]
@abidkh  - Thanks for filing this, but you’ll need to add a CodePen repro before we can help. You can start from one of the [Icon](https://developer.microsoft.com/en-us/fluentui#/controls/web/icon) examples (click “Export to CodePen”) or https://aka.ms/fluentpen.
